README.md

```diff
@@ -177,6 +177,13 @@ converted to a folder, and the license moved into it.
   environment variable `EDITOR` (`nano` and then `vi` are used as fallbacks).
 
   TODO: Describe validation
+- `onyo anchor subfolder`:
+
+  Creates a file  subfolder/.anchor to track a folder even if it has no assets.
+
+- `onyo unanchor subfolder`:
```

Comment:
`onyo ls` seems to be the only command that uses the term `subfolder`. All others use `directory`. We shoudl standardize on `directory`.